beetsplug/mbsync.py

```diff
@@ -134,6 +229,12 @@ def albums(self, lib, query, move, pretend, write):
             # work for albums that have missing or extra tracks.
             mapping = {}
             for item in items:
+                # Clean up obsolete flexible fields
```

Comment:
I admit I don't quite understand why this is necessary… any chance it could have a slightly more detailed explanation? Why do these tags need to be removed?

Reply:
This is because the tags are attributed 'on the fly'. If a performer is tagged as 'Percussions' for example, he will be tagged as 'mbsync_Percussions'. If then someone on MB corrects it to 'Cymbals' mbsync will add a tag 'mbsync_Cymbals' but will not remove the now obsolete 'mbsync_Percussions' tag, unless I remove all tags that start with 'mbsync_' and are not in the new tags. That's why I need this cleanup step.